cmd/gateway/commands.go

```diff
@@ -203,12 +206,15 @@ func createStaticModeCommand() *cobra.Command {
 					LockName: leaderElectionLockName.String(),
 					Identity: podName,
 				},
-				UsageReportConfig:     usageReportConfig,
-				Plus:                  plus,
-				TelemetryReportPeriod: period,
-				Version:               version,
-				ExperimentalFeatures:  gwExperimentalFeatures,
-				ImageSource:           imageSource,
+				UsageReportConfig: usageReportConfig,
+				ProductTelemetryConfig: config.ProductTelemetryConfig{
+					TelemetryReportPeriod:   period,
+					DisableProductTelemetry: disableProductTelemetry,
```

Comment:
I think we should flip this to be `Enabled` to fit the pattern established by HealthConfig, MetricsConfig, and LeaderElection.